helix-core/src/line_ending.rs

```diff
@@ -0,0 +1,65 @@
+use crate::{Rope, RopeGraphemes, RopeSlice};
+
+/// Represents one of the valid Unicode line endings.
+/// VT, FF and PS are excluded here, as we don't expect them to show up as a default line break
```

Comment:
Sorry, my fault for not being clear.  I still think VT, FF, and PS belong in the LineEnding enum.  Specifically, it may be useful in other parts of the code.  For example, maybe when rendering we'll want to re-use this code determining line endings, and render certain line endings differently.  Or something like that.  It also just helps serve to document for other devs what line endings are possible in Unicode, since a lot of people aren't aware of that.

I just meant that VT, FF, and PS probably shouldn't ever be returned by the auto-detect code.